src/sentry/static/sentry/app/components/events/interfaces/generic.tsx

```diff
@@ -34,15 +45,17 @@ export default class GenericInterface extends Component {
     };
   }
 
-  toggleView = value => {
+  state: State;
```

Comment:
Please are you sure that this works? view and data are required..I believe you should set some default value here

Reply:
That works? 🤔 hmm TIL.